lib/plugin.dart

```diff
@@ -388,6 +390,14 @@ class AtomDartPackage extends AtomPackage {
         'type': 'boolean',
         'default': false,
         'order': 11
+      },
+
+      // experimental features
+      'hoverTooltip': {
```

Comment:
We really only need to add config items here if we want them to be exposed to the user. We can still read (`atom.config.getValue()`) arbitrary keys w/o them being listed here.